src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.TextWriter.cs

```diff
@@ -142,8 +142,18 @@ private bool CheckIfWritingJson(string currentContentType)
         internal sealed class ODataUtf8JsonTextWriter : TextWriter
         {
             private readonly ODataUtf8JsonWriter jsonWriter = null;
+            // Buffer used to store chars that could not be encoded due to
+
+            // insufficient data in the input buffer. The chars will be prepended

```

Comment:
nit: Remove empty line in the middle of the statement
```suggestion
            // Buffer used to store chars that could not be encoded due to
            // insufficient data in the input buffer. The chars will be prepended
```